cdap-ui/yarn.lock

```diff
@@ -3620,7 +3590,7 @@ babel-traverse@^6.23.1, babel-traverse@^6.26.0:
     invariant "^2.2.2"
     lodash "^4.17.4"
 
-babel-types@^6.23.0, babel-types@^6.24.1, babel-types@^6.26.0:
+babel-types@^6.23.0, babel-types@^6.26.0:
```

Comment:
was there a change to `package.json`?

Reply:
Yeah we removed the ncc module right?

Reply:
my bad, for some reason the package.json changes was not showing up